src/plugins/kibana_react/public/ui_settings/use_ui_setting.ts

```diff
@@ -60,3 +79,39 @@ export const useUiSetting$ = <T>(key: string, defaultValue?: T): [T, Setter<T>]
 
   return [value, set];
 };
+
+/**
+ * Returns a 2-tuple, where first entry is the setting value and second is a
+ * function to update the setting value.
+ *
+ * Synchronously returns the most current value of the setting and subscribes
+ * to all subsequent updates, which will re-render your component on new values.
+ *
+ * Usage:
+ *
+ * ```js
+ * const [customBranding, setCustomBranding] = useUiSetting$('customBranding:pageTitle');
+ * ```
+ */
+export const useGlobalUiSetting$ = <T>(key: string, defaultValue?: T): [T, Setter<T>] => {
```

Comment:
nit: I'm a bit surprised to see appended `$` 

```suggestion
export const useGlobalUiSetting = <T>(key: string, defaultValue?: T): [T, Setter<T>] => {
```